spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/extractor/TokenBrokerResolver.java

```diff
@@ -98,7 +98,7 @@ public TokenBrokerResolver(XsuaaServiceConfiguration configuration, Cache tokenC
 			AuthenticationInformationExtractor authenticationConfig) {
 		this.configuration = configuration;
 		this.tokenCache = tokenCache;
-		this.tokenBroker = new UaaTokenBroker(tokenService);
+		this.oAuth2TokenService = tokenService;
```

Comment:
@liga-oz can you please check that UaaTokenBroker class is marked as deprecated?

Reply:
@nenaraab yes, it is marked as deprecated